test/cql-pytest/test_guardrail_replication_strategy.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from util import unique_name, config_value_context
+from cassandra.protocol import ConfigurationException
+
+def create_ks_and_assert_warnings_count(cql, ks_opts, warnings_count = 0, fails_count = 0):
+    keyspace = unique_name()
+    if fails_count:
+        with pytest.raises(ConfigurationException):
+            cql.execute_async("CREATE KEYSPACE " + keyspace + " " + ks_opts).result()
+        return
+    else:
+        response_future = cql.execute_async("CREATE KEYSPACE " + keyspace + " " + ks_opts)
+        response_future.result()
+        cql.execute("DROP KEYSPACE " + keyspace)
+        if warnings_count:
+            assert response_future.warnings is not None and len(response_future.warnings) == warnings_count
+        else:
+            assert response_future.warnings is None
+
+def clear_guardrails_config(cql):
+    cql.execute(f"UPDATE system.config SET value = '' where name = 'replication_strategy_warn_list'")
+    cql.execute(f"UPDATE system.config SET value = '' where name = 'replication_strategy_fail_list'")
```

Comment:
I don't understand why you eneded up doing this - clearing the guardrails completely - instead of what I suggested which is to save the old contents and restore it when exiting the test :-( Note that the old context does NOT HAVE TO BE EMPTY - if Scylla's default is not empty (if I understand correctly, the warning list it's not empty in your new code, but defaults to warn about SimpleStrategy, no?).

Reply:
I missed your comment on how to use it in the middle of the test, sorry. I also had to fix `config_value_context` a bit, because it incorrectly handled list values returned from CQL - the solution is rather dumb, but please don't blame me, I spent good amount of time trying more sophisticated approaches and only this one worked for me.